services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/service/implement/SubnetServiceImp.java

```diff
@@ -319,6 +319,26 @@ public boolean verifyCidrBlock(String cidr) throws FallbackException {
 
     }
 
+    @Override
+    @DurationStatistics
+    public Integer getUsedIpByRangeId(String rangeId) throws UsedIpsIsNotCorrect {
+        String ipManagerServiceUrl = ipUrl + "range" + "/" + rangeId;
+        IpAddrRangeRequest ipAddrRange = restTemplate.getForObject(ipManagerServiceUrl, IpAddrRangeRequest.class);
+        if (ipAddrRange == null) {
+            logger.info("can not find ipAddrRange by range id");
```

Comment:
It would be a good practice to always include actual id instead of a static error message.

Reply:
got it